docs/reference/tags/create_line_item.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: default
+title: Create Line Item
+parent: Tags
+has_children: false
+---
+
+The Create Line Item Tag renders a form which acts as a wrapper for a single item.
+
+##### input
+{% raw %}
+```liquid
+    {% form "create_line_item" %}
+        <input type="hidden" data-variant-id="{{item_id}}" name="items[][variant_id]" value="{{item_id}}"/>
+    {% endform %}
+```
+{% endraw %}
+
+##### output
+{% raw %}
+```html
+<form action="/sites/line-items" accept-charset="UTF-8" method="post">
+    <input type="hidden" name="return_to" id="return_to" value="/admin/site_builder/sites/c94b650e/previews/book-tickets" autocomplete="off">
+    <input type="hidden" class="js-variant-input" data-variant-id="BAh7CEkiCGdpZ..." name="items[][variant_id]" value="BAh7CEkiCGdpZAY6Bk....">
```

Comment:
Should we remove the class="js-variant-input" here as that wouldn't be rendered with the above form details?

Reply:
I'll update this now along with the `data-variant-id="BA...` as we're taking this out of the input 🙌